tests/workflow_contracts/ci_lint_test.py

```
@@ -112,6 +112,43 @@ def _steps(workflow: dict[str, object]) -> list[dict[str, object]]:
             raise AssertionError("jobs.build-test.steps must be a list")
 
 
+def _windows_job(workflow: dict[str, object]) -> dict[str, object]:
+    """Return the build-test-windows job."""
+    match workflow.get("jobs"):
+        case dict() as jobs:
+            pass
+        case _:
+            raise AssertionError("the workflow must declare a jobs mapping")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)